doc/doxygen/CMakeLists.txt

```diff
@@ -28,13 +28,6 @@ if(NOT PERL_FOUND)
     "Could not find a perl installation which is required for building the documentation"
     )
 endif()
-if(NOT DOXYGEN_FOUND)
-  message(FATAL_ERROR
-    "Could not find doxygen which is required for building the documentation"
-    )
-else()
-  message(STATUS "Using doxygen version ${DOXYGEN_VERSION}")
-endif()
```

Comment:
We `FATAL_ERROR` here on purpose: This ensures that cmake stops processing and the user actually sees the error message. Otherwise cmake will continue configuring and the actual error tends to get buried. (This is particularly frustrating for all parties when trying to help users on the mailing list. Inexperienced users tend to only tell us the last visible error message `-- Configuring incomplete, errors occurred!`)

Please put the old logic back in place.

Reply:
I also realize how long I have been working on CMake as erroring out with a "fatal error" is now the default for the `REQUIRED` keyword (and had been for a while).

Reply:
Removed it.